File: JavaScript/bee 2137.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

function bubbleSort(arr){
    for(var i = 0; i < arr.length; i++){
        for(var j = 0; j < arr.length; j++){
            if(arr[j] > arr[j + 1]){
                var temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp
            }
        }
    }
    return arr
}

while(n != "eof"){
    var n = lines.shift()
    if (n == undefined){
        break
    }
    if (n.length = 1){
        var contador = n
    } else{
        var codigo = n
    }

    var codigos = []
    for(var i = 0; i < contador; i++){
        var codigo = lines.shift().trim()
        if(codigo.length == 4){
        codigos.push(codigo)
    }
}
    if(codigos[0] != undefined){
        console.log(bubbleSort(codigos).join("\n"))
    }
}

File: C++/bee 2137.cpp
#include <iostream>
#include <iomanip>
#include <stdio.h>
#include <vector>

using namespace std;

void bubbleSort(string arr[], int n){
    for(int i = 0; i < n-1; i++){
        for(int j = 0; j < n-1; j++){
            if(arr[j] > arr[j+1]){
                string temp = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
    }
}

int main()
{
    int n;
    while(cin >> n){
        string arr[n];
        for(int i = 0; i < n; i++){
            string input;
            cin >> input;
            arr[i] = input;
        }
        bubbleSort(arr, n);
        for(int i = 0; i < n; i++){
            cout << arr[i] << endl;
        }
    }
}
